packages/synapse-react-client/test/utils/hooks/SynapseAPI/useEntityBundle.test.tsx

```diff
@@ -25,22 +13,17 @@ const expected: EntityBundle = {
   },
 }
 
-const SynapseClient = require('../../../../src/synapse-client/SynapseClient')
-SynapseClient.getEntityBundleV2 = jest.fn().mockResolvedValue(expected)
+jest.spyOn(SynapseClient, 'getEntityBundleV2').mockResolvedValue(expected)
 
 describe('useEntityBundle functionality', () => {
-  beforeEach(() => {
-    queryClient.clear()
-  })
-
   it('correctly calls SynapseClient', async () => {
     const entityId = 'syn123'
 
-    const { result, waitFor } = renderHook(() => useGetEntityBundle(entityId), {
-      wrapper,
+    const { result } = renderHook(() => useGetEntityBundle(entityId), {
+      wrapper: createWrapper(),
     })
 
-    await waitFor(() => result.current.isSuccess)
```

Comment:
The returned `waitFor` must have allowed returning a boolean. The imported `waitFor` check doesn't work unless an error is thrown, such as if an `expect` fails.